graphql/schema/types/image.graphql

```diff
@@ -0,0 +1,12 @@
+###### TYPES ######
+"""
+Image is returned by the image query.
+It contains information about an image.
+"""
+type Image {
+  name: String!
+  versionId: String!
+  kernel: String!
+  lastDeployed: Time!
+  ami: String!
+}
```

Comment:
nit. files should end in a new line. There's a VSCode setting for this you can enable called `Insert final newline` which will do this for you automatically!